src/Servers/Kestrel/Core/src/Internal/Http3/Helpers/VariableIntHelper.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Diagnostics;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3
+{
+    public static class VariableIntHelper
+    {
+        public static long GetVariableIntErrorIfNotAllowedValue(in ReadOnlySequence<byte> buffer, out SequencePosition consumed, out SequencePosition examined)
```

Comment:
What does this mean? The comments below refer to reserved stream ids.
I don't see it used anywhere.

Reply:
Adding a doc comment.